pkg/services/sqlstore/migrator/sqlite_dialect_cgo.go

```diff
@@ -0,0 +1,47 @@
+//go:build cgo
+
+package migrator
+
+import (
+	"errors"
+
+	"github.com/mattn/go-sqlite3"
+)
+
+const SQLite = "sqlite3"
+
+var SQLite3Driver = &sqlite3.SQLiteDriver{}
+
+func IsSQLiteErrLocked(err error) bool {
+	return errors.Is(err, sqlite3.ErrLocked) || errors.Is(err, sqlite3.ErrBusy)
+}
+func SQLiteErrCode(err error) int {
+	var sqliteErr sqlite3.Error
+	if !errors.As(err, &sqliteErr) {
+		return 0
+	}
+	return int(sqliteErr.Code)
```

Comment:
What is the difference between Code here and ExtendedCode on line 29 (and in the existing code)?